test/integration/ratio-Encounter-reuseObservationFunction/patients/patient-bundle.json

```diff
@@ -0,0 +1,184 @@
+{
```

Comment:
I wonder if for consistency we should create a new patient in fqm-testify just so the format of patient data is consistent across all integration tests?

Reply:
Did this in f207793. Want to add another test to better test it after I get the shared function for checking observ results done.